packages/connect-fastify/src/fastify-connect-plugin.ts

```diff
@@ -60,30 +60,21 @@ export function fastifyConnectPlugin(
     done();
     return;
   }
+  if (opts.acceptCompression === undefined) {
+    opts.acceptCompression = [compressionGzip, compressionBrotli];
+  }
   const router = createConnectRouter(opts);
   opts.routes(router);
+
   const uHandlers = router.handlers;
   if (uHandlers.length == 0) {
     done();
     return;
   }
 
-  if (opts.acceptCompression === undefined) {
-    opts.acceptCompression = [compressionGzip, compressionBrotli];
-  }
-
-  // we can override all content type parsers (including application/json) in this plugin
-  // without affecting outer scope
-  const types = [
-    protoConnect.contentTypeRegExp,
-    protoGrpcWeb.contentTypeRegExp,
-    protoGrpc.contentTypeRegExp,
-  ];
-  for (const type of types) {
-    instance.addContentTypeParser(type, (_req, _payload, done) => {
-      done(null, undefined);
-    });
-  }
+  // we can override all content type parsers (including application/json) in
+  // this plugin without affecting outer scope
+  addNoopContentTypeParsers(instance);
```

Comment:
We are adding all known content-types as string _and_ as regex now. From the [fastify docs](https://www.fastify.io/docs/latest/Reference/ContentTypeParser/):

> Fastify first tries to match a content-type parser with a string value before trying to find a matching RegExp.